### ISSUE TITLE
Comfortaa rendering issues in several applications due to overlaps

### ISSUE THREAD

**Issue by V17** [google/fonts#4818]
I've been using Comfortaa for some time and several times I noticed weird issues that looked like broken antialiasing on some glyphs, as if parts of the glyph were a subpixel amount wider. 

This issue only applies to the Google fonts version. Previous versions of the font from dafont, font squirrel or author's Deviant Art page do not have this problem.

I think I have found the reason - some of the glyphs have overlaps that to my layman eye look nonsensical and obviously some rendering engines cannot deal with them.

This is most visible when you load the font into Blender, a 3D modelling application that allows you to thicken the font using an "offset" parameter. Some of the overlapping components that the glyph is made from correctly thicken and others incorrectly thin at the same time, as if the application incorrectly labels which part of the component is outside and which is inside.

Standard behaviour in Blender is that each glyph is visibly made from just one component, so not only do the glyphs thicken or thin correctly, but there are no visible boundaries inside each glyph at all. Comfortaa is the only font I've had this issue with. 

The antialiasing issues in other applications occur at the same spots where these overlap issues happen in Blender. I'm currently unable to reproduce it in a web browser (I think it used to be broken in Firefox), so that could have been fixed, but it still happens in Affinity Suite for example.

I first noticed it with "h", so I'm using it as an example.

Affinity Publisher: 
![hlook antialiasing](https://user-images.githubusercontent.com/16502808/174806032-7edc63da-c929-493e-8c42-8c821f49a475.png)

Blender examples: 

Thickening with "offset" parameter
![blender rendering-offset-thickening](https://user-images.githubusercontent.com/16502808/174806162-c13ba697-76f9-46b9-aceb-d2d786eb6399.png)¨

Extrusion to better show the individual components of the glyphs
![blender rendering extrude](https://user-images.githubusercontent.com/16502808/174806171-770e8ed0-a3ca-404c-82ea-b196239998f4.png)

Standard flat rendering is also broken
![blender rendering flat](https://user-images.githubusercontent.com/16502808/174806176-ed0a7113-9653-45f5-90c7-a14d835ea2da.png)

**Comment by RosaWagner** [comment]
Yes, this is not a bug. GF decided to keep overlapping outlines in the static fonts generated from a variable font. If you use an app not supporting that, you need to download the static fonts from the upstream repository and install them locally :)

**Comment by lynrayy** [comment]
Still relevant
